packages/stark-ui/src/modules/route-search/components/_route-search-theme.scss

```diff
@@ -5,7 +5,7 @@
 .search-field {
   .mat-form-field-wrapper {
     border: 1px solid mat-color($grey-palette, 400);
-    background-color: #ffff;
+    background-color: $offwhite;
```

Comment:
Could you please put back `#fff` ? 😊 
Because the background-color of the auto-complete is white too. Otherwise, it's a bit weird according to me...

Reply:
I updated the PR. 😄